code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -206,6 +207,22 @@ GLOBAL_LIST_INIT(nonreactive_gases, typecacheof(list(/datum/gas/oxygen, /datum/g
 	moles_visible = MOLES_GAS_VISIBLE
 	rarity = 300
 
+/datum/gas/helium
+	id = "helium"
+	specific_heat = 15
+	name = "Helium"
+	dangerous = FALSE
+	fusion_power = 7
+	rarity = 50
+
+/datum/gas/antinoblium
+	id = "antinoblium"
+	specific_heat = 1
```

Comment:
Yes and no, need more testing

Reply:
any new thoughts?

Reply:
I think i'll leave it as is now, since this specific_heat has purpose inside the HFR (easier to heat up gas, more energy, etc)
also i was thinking of not adding much more functions to both the He and Antinob in this PR, since i was already planning other PRs to work with this one that will add functions and purposes to these gases.